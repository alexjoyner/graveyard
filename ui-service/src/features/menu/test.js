import React from 'react';
import { shallow } from 'enzyme';
import { Menu } from '.';

describe('`Menu`', () => {
    let component;
    beforeEach(() => {
        component = shallow(<Menu />);
    });
    it('reanders without exploding', () => {
        expect(component).toMatchSnapshot();
    });
});