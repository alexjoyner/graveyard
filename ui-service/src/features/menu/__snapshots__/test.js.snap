// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Menu\` reanders without exploding 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Menu />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Menu
</h1>,
      "id": "Menu",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Menu",
      },
      "ref": null,
      "rendered": "Menu",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Menu
</h1>,
        "id": "Menu",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Menu",
        },
        "ref": null,
        "rendered": "Menu",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
